tdameritrade/client.py

```diff
@@ -116,3 +116,8 @@ def movers(self, index, direction='up', change_type='percent'):
                             headers=self._headers(),
                             params={'direction': direction,
                                     'change_type': change_type}).json()
+
+    def orders(self, account_id, order):
+        return requests.post(ACCOUNTS + account_id + "/orders",
+                             headers=self._headers(),
+                             json=order).json()
```

Comment:
You need to remove `.json()` in here. It only works if the request get an error code. If there is no error and request returned with code 201, there is nothing to convert it to by `json()`.

Reply:
Good catch! I’ll fix it in a bit